package.json

```diff
@@ -11,26 +11,24 @@
     "dateformat": "1.0.6-1.2.3",
     "debug": "*",
     "download": "0.1.7",
-    "diveSync": "0.2.1",
     "ecstatic": "0.5.4",
     "ejs": "0.8.4",
     "empty-port": "~0.0.1",
     "fe.js": "~0.1.4",
     "ffi-prebuilt": "~1.2.6",
-    "gaia-raptor": "0.2.0",
+    "gaia-raptor": "0.6.0",
     "grunt": "0.4.2",
     "grunt-contrib-clean": "0.5.0",
     "grunt-jsdoc": "0.5.1",
     "istanbul": "^0.3.2",
-    "jsdom-nogyp": "^0.8.3",
-    "jshint": "2.5.10",
+    "jshint": "2.6.0",
```

Comment:
looks like the new jshint is not really happy with some code, see https://treeherder.mozilla.org/logviewer.html#?job_id=546130&repo=gaia-try